mycore-base/src/main/java/org/mycore/frontend/ws/endoint/MCRAbstractEndpoint.java

```diff
@@ -32,15 +32,18 @@
 
 /**
  * Starting class for all mycore websocket endpoints.
- * 
+ *
  * @author Michel Buechner (mcrmibue)
  * @author Matthias Eichner
  */
-public abstract class MCRAbstractEndpoint {
+public class MCRAbstractEndpoint {
```

Comment:
The class name could may be confusing.

Reply:
Yeah I noticed that and second this issue. Meanwhile I want to merge it to 2024.06.x and there should be no incompatible API change. So we should definitively address this issue in another PR.